OSVR-Unity/Assets/OSVRUnity/Resources/OsvrDistortion.shader

```diff
@@ -18,66 +18,59 @@
 /// limitations under the License.
 /// </copyright>
 /// <summary>
-/// Based on Unity's (Pro Only) Image Effect: Fisheye.shader
+/// Long ago, based on Unity's (Pro Only) Image Effect: Fisheye.shader
```

Comment:
Are we sure that we can use this under the Apache 2.0 or do we need a clean-room implementation, given that it's based on an old unity shader?